rootfs/rootfs/etc/vmware-tools/scripts/vmware/network

```diff
@@ -0,0 +1,596 @@
+#!/bin/sh
```

Comment:
It is indeed needed by vmtoolsd, I will add it to the patch routine instead of adding the file directly to the rootfs.